druid/src/widget/image.rs

```diff
@@ -60,18 +57,18 @@ impl<T: Data> Image<T> {
     }
 }
 
-impl<T: Data> Widget<T> for Image<T> {
-    fn event(&mut self, _ctx: &mut EventCtx, _event: &Event, _data: &mut T, _env: &Env) {}
+impl Widget<u32> for Image {
```

Comment:
here we do need a `T`.

The rationale is that the image might need to go in a container, which is part of a widget graph that has some data type.

So it's fine to remove `T` from `Image`, but we still need `impl<T: Data> Widget<T> for Image`.